src/calendar/day.jsx

```diff
@@ -3,26 +3,37 @@ import formatDate from 'date-fns/format'
 import PropTypes from 'prop-types'
 import React from 'react'
 
-const Day = ({
-  blockClassName,
-  date,
-  handleOnClick,
-  handleOnEnter,
-  isCurrentMonth,
-  isDisabled,
-  isHighlighted,
-  isMonthNext,
-  isMonthPrev,
-  isNonSelectable,
-  isSelectable,
-  isSelected,
-  isSelectionEnd,
-  isSelectionStart,
-  isToday,
-  isWeekend,
-  isWorkday,
-}) => (
-  <button
+const Day = (props) => {
+  const {
+    blockClassName,
+    date,
+    handleOnClick,
+    handleOnEnter,
+    isCurrentMonth,
+    isDisabled,
+    isHighlighted,
+    isMonthNext,
+    isMonthPrev,
+    isNonSelectable,
+    isSelectable,
+    isSelected,
+    isSelectionEnd,
+    isSelectionStart,
+    isToday,
+    isWeekend,
+    isWorkday,
+    customRender
+  } = props
+  const children = formatDate(date, 'D')
+
+  if (customRender) {
+    return customRender({
+      ...props,
+      children
+    })
```

Comment:
I would pass the date only without format, because it can be handled on the passed render prop depend on the usecase

Reply:
date is passed here as a part of props, as for children, I wanted to keep backward compatibility, that's the only reason